oidc-controller/api/core/config.py

```diff
@@ -1,12 +1,98 @@
+import json
 import logging
+import sys
+import logging.config
+import structlog
 import os
 from enum import Enum
 from functools import lru_cache
 from typing import Optional, Union
 
 from pydantic import BaseSettings
 
-logger = logging.getLogger(__name__)
+from pathlib import Path
+
+logging.basicConfig(
+    format="%(message)s",
+    stream=sys.stdout,
+    level=logging.DEBUG,
+)
+
+# Use environment variable to determin logging format
```

Comment:
Nit.: typo `determine`

Reply:
Logging level setting should be exposed via an ENVVAR as well.